prep-javascript-variables/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Marianne Cuevas';
+    console.log(fullName);
+    var numberofPets = '0';
```

Comment:
For numbers, we'll want to leave off the quotes so that the JavaScript engine doesn't interpret them as strings.